app/lib/pre_assembly/batch.rb

```diff
@@ -134,41 +152,7 @@ def process_digital_objects
     # rubocop:enable Metrics/AbcSize
     # rubocop:enable Metrics/MethodLength
     # rubocop:enable Metrics/CyclomaticComplexity
-
-    # @return [Array<DigitalObject>]
-    def objects_to_process
-      @objects_to_process ||= digital_objects.reject { |dobj| skippables.key?(dobj.container) }
-    end
-
-    private
-
-    def stager
-      staging_style_symlink ? LinkStager : CopyStager
-    end
-
-    # Discover object containers from the object manifest file suppled in the bundle_dir.
-    def discover_containers_via_manifest
-      manifest_rows.each_with_index do |r, i|
-        next if r[:object]
-        raise 'Missing header row in manifest.csv' if i == 0
-
-        raise "Missing 'object' in row #{i}: #{r}"
-      end
-      manifest_rows.map { |r| bundle_dir_with_path r[:object] }
-    end
-
-    # A method to discover stageable items (i.e. files) with a given object folder.
-    # Takes a root path of the object folder (as supplied in the object manifest).
-    # It then finds all files within with an eager glob pattern.
-    def discover_items_via_crawl(root)
-      Dir.glob("#{root}/**/*")
-    end
-
-    # A convenience method to return all ObjectFiles for all digital objects.
-    # Also used for stubbing during testing.
-    def all_object_files
-      digital_objects.map(&:object_files).flatten
-    end
```

Comment:
git thinks this code was removed, but it got confused by the methods I moved to private.  I didn't touch this code.